elide-datastore/elide-datastore-hibernate/src/test/java/com/yahoo/elide/datastores/hibernate/hql/AbstractHQLQueryBuilderTest.java

```diff
@@ -54,15 +55,21 @@
     private static final String DEF = "DEF";
     private static final String NAME = "name";
 
+
     public AbstractHQLQueryBuilderTest() {
-        super(new EntityDictionary(new HashMap<>()), new TestSessionWrapper());
+        super(getMockEntityProjection(), new EntityDictionary(new HashMap<>()), new TestSessionWrapper());
         dictionary.bindEntity(Author.class);
         dictionary.bindEntity(Book.class);
         dictionary.bindEntity(Chapter.class);
         dictionary.bindEntity(Publisher.class);
         dictionary.bindEntity(Left.class);
     }
 
+    private static  EntityProjection getMockEntityProjection() {
```

Comment:
Do we need a method for this?

Reply:
I had to create this method, due to the fact that calling super should be the first statement of the constructor, where this method is used.